_rules/6cbcbe.md

```diff
@@ -0,0 +1,206 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that identical accessible names are only used for links that serve an equivalent purpose
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [included in the accessibility tree](#included-in-the-accessibility-tree) and that have the same [non-empty](#non-empty) [accessible name](#accessible-name).
+
+### Expectation
+
+When activated, the links in each set of target elements resolve to resources that fulfill an [equivalent purpose](#equivalent-purpose) in relation to the [accessible names](#accessible-name) of the link.
+
+**Note:** Resolving the links includes potential redirects.
+
+## Assumptions
+
+* This rule assumes that the purpose of the links for links with identical names would not be ambiguous to users in general.
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding Success Criterion 2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)
+- [URL parsing](#https://www.w3.org/TR/html52/infrastructure.html#parsing-urls)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+Identical link text leads to identical URLs:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+```
+
+#### Passed example 2
+
+Links resolves to same page after redirect:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/redirect.html">Contact us</a>
+```
+
+#### Passed example 3
+
+Identical pages are located on different URLs:
```

Comment:
Isn't this covered in the previous test case in that both the same page located somewhere else and a redirected page serve up content of an equivalent purpose?

Reply:
I still think they are different conceptually.